### PR TITLE
Add `wp-image-####` CSS class to images in the Media and Text block

### PR DESCRIPTION
**Pull request by azaozz**
This is used to add `srcset` and `sizes` attributes on the front-end. As we use the "large" image size now, having these attributes becomes (almost) mandatory :)

### CODE REVIEW

File: packages/block-library/src/media-text/index.js [hunk 1]
```diff
@@ -152,12 +154,13 @@ export const settings = {
 			mediaType,
 			mediaUrl,
 			mediaWidth,
+			mediaId,
 		} = attributes;
 		const mediaTypeRenders = {
-			image: () => <img src={ mediaUrl } alt={ mediaAlt } />,
+			image: () => <img src={ mediaUrl } alt={ mediaAlt } className={ ( mediaId && mediaType === 'image' ) ? `wp-image-${ mediaId }` : null } />,
```
Comment (mtias) [inline packages/block-library/src/media-text/index.js]: Do we need a deprecation just for the `className`?

Comment (mtias) [reply packages/block-library/src/media-text/index.js]: cc @aduth

**Comment by aduth** [comment]
>This is used to add srcset and sizes attributes on the front-end.

Can't this be determined via `mediaId`?

**Comment by azaozz** [comment]
> Can't this be determined via mediaId?

Yes, this is pretty much the same as in the image block. When there is a `mediaId` we add the `wp-image-####` CSS class to the img tag so it's picked by `wp_image_add_srcset_and_sizes()` on display.

File: packages/block-library/src/media-text/index.js (hunk 1 above)
Comment (jorgefilipecosta) [reply packages/block-library/src/media-text/index.js]: In this case given that the class being added is an internal one we need the deprecations.

**Review by jorgefilipecosta — APPROVED** [review]
I tested this changes and it looks like it works as expected, thank you for this enhancement @azaozz. On the frontend I'm now getting sizes and srcset on images added on media text block.
I added a commit that updates our text fixtures, I think this is ready.

File: packages/block-library/src/media-text/index.js [hunk 2]
```diff
@@ -152,12 +154,13 @@ export const settings = {
...
 			video: () => <video controls src={ mediaUrl } />,
-		};
 
+		};
```
[the remark this answers is not in the document]
Comment (azaozz) [reply packages/block-library/src/media-text/index.js]: Uh, sorry.

**Comment by azaozz** [comment]
Thanks! Looks great. Was going to fix the tests tonight but got sidetracked a bit.

**Pull request merged**